devito/ir/clusters/cluster.py

```diff
@@ -540,19 +544,3 @@ def reduce_properties(clusters):
             properties[d] = normalize_properties(properties.get(d, v), v)
 
     return Properties(properties)
-
-
-def tailor_properties(properties, ispace):
```

Comment:
note for reviewers: finally moved to ir/support/properties as promised in an older PR